docs/ref/contrib/postgres/indexes.txt

```diff
@@ -33,4 +33,15 @@ available from the ``django.contrib.postgres.indexes`` module.
     <https://www.postgresql.org/docs/current/static/btree-gin.html>`_ on
     PostgreSQL. You can install it using the
     :class:`~django.contrib.postgres.operations.BtreeGinExtension` migration
-    operation.
+    operation. The parameter ``fastupdate`` takes a boolean value (default in
```

Comment:
Is there PostgreSQL docs we can link to that explains these options? As something unfamiliar with GIN indexes, I'm complete at a loss for how I'd use these options. Also, the `test_gin_parameters` is failing on PostgreSQL < 9.5 since `gin_pending_list_limit` isn't supported there.